app/controllers/users_controller.rb

```diff
@@ -51,7 +51,7 @@ def reorder_call_list
   private
 
   def user_params
-    params.require(:user).permit(:name, :email, :_role)
+    params.require(:user).permit(:name, :email, :role)
```

Comment:
Similar to earlier question, why from _role to role? I assume it's a difference between the two libs, just want to understand.